### ISSUE TITLE
[BUG] hasAnnotationReader Type Changed

### ISSUE THREAD

**Issue by aftabnaveed** [laravel-doctrine/orm#476]
Please prefix your issue with one of the following: [BUG] [PROPOSAL] [QUESTION].

### Package version, Laravel version 
Laravel 8.0

### Actual behaviour
```
  Argument 1 passed to LaravelDoctrine\Extensions\GedmoExtensionsServiceProvider::hasAnnotationReader() must be an instance of Doctrine\Common\Persistence\Mapping\Driver\MappingDriverChain, instance of LaravelDoctrine\ORM\Extensions\MappingDriverChain given, called in /home/naveed/Projects/test/vendor/laravel-doctrine/extensions/src/GedmoExtensionsServiceProvider.php on line 26
```
After running composer update on my existing project, it looks like some of the updates w.r.t GedmoExtension broke my project. Is it just me or anyone else is facing a similar issue?

**Comment by eigan** [comment]
This was broken when we changed from Doctrine\Common\Persistence to Doctrine\Persistence here in the ORM project. 

Released 1.3.1 of laravel-doctrine/extensions now which I think should fix it. Could you try it out?

Commit:
https://github.com/laravel-doctrine/extensions/commit/d2dc2d6f438ff992f54a648dbba09369f7beaf76

**Comment by aftabnaveed** [comment]
This is now fixed. 

Thanks
